src/main/app/src/Forms/CustomerForm.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+
+
+export default function AddCustomerForm() {
```

Comment:
The dialog looks great!

***One suggestion:*** 
This component, `AddCustomerForm`, is almost identical to the `FormDialog` component from src/main/app/src/Forms/EditForm.js. The best way to handle this would be to make a reusable component that displays this dialog, and pass in properties for customizing the form dialog. Basically a component that you can call like:

```javascript
<GenericCustomerDialog 
  dialogTitle='Customized Title' 
  dialogContentText='Some custom dialog content text' 
  submitButtonText='Save or Update'  
/>
```

You can pass in the properties to your function via the function parameter `props` 

```javascript
function GenericCustomerDialog(props) {

  // useState hook calls
  // Handler function definitions

  return (
    <div>
      <Button variant="contained" color="primary" style={{ float: 'right' }} onClick={handleClickOpen}>

      // You can access elements of the props object anywhere in your function
      {props.dialogTitle}
      </Button>
      // ...
}
```

Reply:
Thanks I will work on it!